Makefile

```diff
@@ -136,17 +136,16 @@ ifdef K8S_SFX_AGENT
 endif
 .PHONY: run-k8s-tests
 run-k8s-tests:
-	docker rm -f -v minikube || true
 	pytest \
 		-m "k8s" \
-		-n 4 \
+		-n auto \
 		--verbose \
 		--exitfirst \
 		--k8s-observers=k8s-api,k8s-kubelet \
 		--html=test_output/k8s_results.html \
 		--self-contained-html \
-		$(agent_image_arg) $(k8s_version_arg) tests || true
-	docker rm -f -v minikube
+		$(agent_image_arg) $(k8s_version_arg) tests && \
+	docker rm -fv minikube
```

Comment:
I would put some kind of `echo` output saying that minikube has been left running since the tests failed or something to at least give the user a bit more feedback.

Reply:
Echo `minikube container is left running for debugging purposes` if tests fail; otherwise remove the minikube container.